webui/react/src/pages/ExperimentDetails/ExperimentDetailsHeader.tsx

```diff
@@ -23,14 +24,20 @@ import { openCommand } from 'wait';
 import css from './ExperimentDetailsHeader.module.scss';
 
 interface Props {
+  continueTrialRef: React.RefObject<ContinueTrialHandles>;
```

Comment:
**suggestion:** What do you think about following the same pattern as `showForkModal: () => void;` instead of passing in a ref?

Reply:
I used the ref to be consistent, but that's good to. Changed.